File: src/main/java/ti/proyectoinia/dtos/SanitarioDto.java
package ti.proyectoinia.dtos;

import lombok.Data;

import java.util.List;

@Data
public class SanitarioDto {

    private Long id;

    private String fechaSiembra;

    private String fecha;

    private String metodo;

    private Integer temperatura;

    private Integer horasLuz;

    private Integer horasOscuridad;

    private Integer nroDias;

    private String estado;

    private String observaciones;

    private Integer nroSemillasRepeticion;

    private Long reciboId;

    private boolean activo;

    private boolean estandar;

    private boolean repetido;

    private List<Long> sanitarioHongosId;

    private String fechaCreacion;

}

File: frontend/src/models/Sanitario.dto.ts
export interface SanitarioDto {
  id: number | null;
  fechaSiembra: string | null;
  fecha: string | null;
  metodo: string;
  temperatura: number | null;
  horasLuz: number | null;
  horasOscuridad: number | null;
  nroDias: number | null;
  estado: string;
  observaciones: string;
  nroSemillasRepeticion: number | null;
  reciboId: number | null;
  activo: boolean;
  estandar: boolean;
  repetido: boolean;
  sanitarioHongoids: number[] | null;
  fechaCreacion: string | null;
  fechaRepeticion: string | null;
}
